01_fib/pfannerj/main_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	mainout = &buf
+
+	main()
+
+	expected := strconv.Quote("Fibonacci series starting...\nFibonacci series completed...\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestFibOutputFibZero(t *testing.T) {
+	var buf bytes.Buffer
+	fibout = &buf
+
+	fib(0)
+
+	expected := strconv.Quote("")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in fib(0)")
+	}
+}
+
+func TestFibOutputFibSeven(t *testing.T) {
+	var buf bytes.Buffer
+	fibout = &buf
+
+	fib(7)
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
```

Comment:
Again this fine, but to help readability a little I did something like this:


```
        numbers := []string{
                "1",
                "1",
                "2",
                "3",
                "5",
                "8",
                "13",
        }
        expected := strings.Join(numbers, "\n") + "\n"
```

Reply:
Alternatively you could use raw strings:
expected := `1
1
2
3
5
8
13
"

Reply:
Since I need to do a 'rebase' (as per Julia's comments) I will include this change...